package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@actions/core": "^1.2.6",
     "@actions/tool-cache": "^1.6.1",
+    "del": "^6.0.0",
```

Comment:
I'm meaning to remove `make-dir` and `mkdirp-promise` and replace with `@actions/io`. Would love for you to use [`rmRF`](https://github.com/actions/toolkit/tree/main/packages/io#rm--rf) instead of `del` :)

Reply:
Sure. I'm unfamiliar with the actions eco system. Happy to replace this